tests/unit_tests/test_platzky.py

```
@@ -85,18 +78,44 @@ def test_create_app(self):
+        mock_config.feature_flags = {"FAKE_LOGIN": True}
+
+        app = create_engine(mock_config, mock_db)
+
+        app.secret_key = "test_secret_key"
+        client = app.test_client()
+
+        with app.test_request_context():
+            response = client.get("/admin/fake-login/admin")
+            assert response.status_code == 301
+            with client.session_transaction() as sess:
+                import pdb; pdb.set_trace()
```

Ruff (0.8.2)

Multiple statements on one line (semicolon)

(E702)